packages/jest-jasmine2/src/index.js

```diff
@@ -69,7 +69,49 @@ function jasmine2(config, environment, moduleLoader, testPath) {
       moduleLoader.requireModule(null, config.setupTestFrameworkScriptFile);
     }
   });
+
+  const hasIterator = object => !!(object !== null && object[Symbol.iterator]);
```

Comment:
This should be `object != null` so that undefined is also ignored. `undefined !== null` is true.